protocol/communities/manager.go

```diff
@@ -481,6 +481,11 @@ func (m *Manager) Start() error {
 	if m.ownerVerifier != nil {
 		m.runOwnerVerificationLoop()
 	}
+
+	go func() {
+		_ = m.fillMissingCommunityTokens()
```

Comment:
Just wondering if it makes sense to log something in case of error since `fillMissingCommunityTokens` returns an error? Did you maybe anticipate this might be too verbose?

Reply:
I do all the logs directly in `fillMissingCommunityTokens`. I could add an additional log here, but it would just repeat the error and lack the additional context.